hw/ip/pattgen/data/pattgen.prj.hjson

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+    name:               "pattgen",
+    design_spec:        "hw/ip/pattgen/doc",
+    dv_plan:            "hw/ip/pattgen/doc/dv_plan",
+    hw_checklist:       "hw/ip/pattgen/doc/checklist",
```

Comment:
These documents don't exist yet (and that's fine). Can you comment them out until the documents exists?
```suggestion
//    dv_plan:            "hw/ip/pattgen/doc/dv_plan",
//    hw_checklist:       "hw/ip/pattgen/doc/checklist",
```

Reply:
Will comment them out if they aren't already in there.

Reply:
Will be commented out in next commit